File: web/src/api/normal/gost_obs.js
import request from "../../request/index.js";

const baseUrl = '/v1/normal/gost/obs'

export const apiNormalGostObsTunnelMonth = (data) => {
    return request.request({
        url: `${baseUrl}/tunnel/month`,
        method: 'POST',
        data
    })
}

export const apiNormalGostObsClientMonth = (data) => {
    return request.request({
        url: `${baseUrl}/client/month`,
        method: 'POST',
        data
    })
}

export const apiNormalGostObsNodeMonth = (data) => {
    return request.request({
        url: `${baseUrl}/node/month`,
        method: 'POST',
        data
    })
}

export const apiNormalGostObsUserMonth = (data) => {
    return request.request({
        url: `${baseUrl}/user/month`,
        method: 'POST',
        data
    })
}

File: server/router/normal/gost_obs.go
package normal

import (
	"github.com/gin-gonic/gin"
	"server/controller/normal/gost_obs"
	"server/global"
	"server/router/middleware"
)

func InitGostObs(group *gin.RouterGroup) {
	g := group.Group("gost/obs", middleware.Auth(global.Jwt))
	g.POST("tunnel/month", gost_obs.TunnelMonth)
	g.POST("client/month", gost_obs.ClientMonth)
	g.POST("node/month", gost_obs.NodeMonth)
	g.POST("user/month", gost_obs.UserMonth)
}
